tests/test_inout.py

```diff
@@ -68,3 +69,39 @@ def __call__(self, x):
     def test_implicit_input(self):
         x = chainer.Variable(np.array(1, dtype=np.float32))
         test_onnxruntime.check_output(self.model, x, self.fn)
+
+
+@testing.parameterize(
+    {'use_bn': True},
+    {'use_bn': False},
+)
+class TestMultipleOutput(unittest.TestCase):
+
+    def get_model(self, use_bn=False):
+        class Model(chainer.Chain):
+
+            def __init__(self, use_bn=False):
+                super(Model, self).__init__()
+
+                self._use_bn = use_bn
+                with self.init_scope():
+                    self.conv = L.Convolution2D(None, 32, ksize=3, stride=1)
+                    if self._use_bn:
+                        self.bn = L.BatchNormalization(32)
+
+            def __call__(self, x):
+                h = self.conv(x)
+                if self._use_bn:
+                    h = self.bn(h)
+                return {
```

Comment:
Perhaps it's worth adding a testcase which returns a tuple of Variables?

Reply:
thx, add tuple and list test case